bin/common.ml

```diff
@@ -534,6 +534,7 @@ module Builder = struct
     ; dump_memo_graph_file : string option
     ; dump_memo_graph_format : Graph.File_format.t
     ; dump_memo_graph_with_timing : bool
+    ; dump_gc_stats : string option
```

Comment:
You can use `Path.External.t option` here and convert it with `Path.External.of_filename_relative_to_initial_cwd`.